index.html

```diff
@@ -554,13 +520,9 @@ <h2>
             When called, this method executes the following steps:
           </p>
           <ol>
-            <li>Let <var>permission</var> be the result of running
-              <a data-cite="!permissions#dfn-retrieve-the-permission-state">retrieving
-              the permission state</a> of the permission associated with
-              <a>payment handler</a>'s <a>origin</a>.
-            </li>
-            <li>If <var>permission</var> is not "granted", then return a
-            <a>Promise</a> rejected with a <a>NotAllowedError</a>.
+            <li>Upon user agent discretion and depending on user consent,
```

Comment:
This feels like a side-effect, but maybe it isn't. I guess we need to describe this further as to how user consent is acquired.